doc/source/user_guide/make_container_image.rst

```diff
@@ -0,0 +1,108 @@
+.. _ref_make_container_image:
+
+Containerization of Fluent
+==========================
+
+.. warning:: You need a valid Ansys license to follow the steps in this section.
+
+This document provides instructions and guidelines on how to containerize
+Fluent for efficient and secure deployment and management.
+
+
+Prerequisites
+-------------
+
+* A Linux machine with `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys license. Your Ansys reseller should have provided you with one.
+
+* The following provided files:
+  
+  * `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_
+  * `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_
+
+
+Procedure
+---------
+
+These text files indicate the files that are included or excluded during the copying for Ansys version 24R1.
+If you have a different version of Ansys then update these paths in the following text files accordingly, for example if you have
+the pre-installed Ansys version 23R2 then replace v241 with v232 and 24.1.0 with 23.2.0 etc.
```

Comment:
_etc_ at the end is too vague to be included in instructions. take this list:
```
v241/CEI/BUILDINFO.txt
v241/CEI/apex241
v241/CEI/bin
v241/CEI/ensight241lang
v241/CEI/nexus241
```
Those embedded `241`s must also be updated, but are tey covered by the etc or not? - as a user, I would have a doubt.

Reply:
Maybe, we can publish separate filelists for each release. Files can be added/deleted in newer version, e.g.

Reply:
Removed. Thank you.